docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -309,6 +309,18 @@ focus for this specification. More specifically, on non-EFI systems
 configuration snippets following this specification cannot be used to spawn
 other operating systems (such as Windows).
 
+Unfortunately, there are implementations of boot loading infrastructure that
+are also using the /loader/entries/ directory, but place files in them that are
+not valid by this specification. In order to minimize confusion a boot loader
+implementation may place a file /loader/entries.srel next to the
```

Comment:
I forgot to ask this: why "srel"? Is this short for something?

Reply:
well, i didn't want to call it "standard", because that was already he string inside it. So I wanted to name it something that tries to communicate "which standard?". So I came up with "srel", i.e. the "standard release"... choose from fedora standard release, or systemd standard release...

But then i wanted it a bit cryptic so that it can mean anything or nothing.

Anyway, not glued to that name. if you have a better idea, i am game.

Reply:
Oh, OK. I guessed so, but I thought that maybe "srel" is already used somewhere. I guess the extension doesn't matter, as long as it's set to something.